docs/software_requirements/interrupts.sdoc

```diff
@@ -10,53 +10,194 @@ UID: ZEP-SRS-7-1
 STATUS: Draft
 TYPE: Functional
 COMPONENT: Interrupts
-TITLE: Service routine for handling interrupts (ISR)
+TITLE: Installing static IRQ service routines (ISR).
 STATEMENT: >>>
-The Zephyr RTOS shall provide support a service routine for handling interrupts (ISR).
+Zephyr RTOS shall provide a mechanism to initialize a static IRQ service routine (ISR),
+providing all parameters needed to configure the hardware and software.
 <<<
+RELATIONS:
+- TYPE: Parent
+  VALUE: ZEP-39
```

Comment:
Link need to be adapted

Reply:
updated all the ZEP-39 tags to ZEP-SYRS-7